priam/src/main/java/com/netflix/priam/tuner/StandardTuner.java

```diff
@@ -63,11 +67,13 @@ public void writeAllProperties(String yamlLocation, String hostname, String seed
         map.put("native_transport_port", config.getNativeTransportPort());
         map.put("listen_address", hostname);
         map.put("rpc_address", hostname);
+
         // Dont bootstrap in restore mode
-        if (!Restore.isRestoreEnabled(config, instanceInfo)) {
-            map.put("auto_bootstrap", config.getAutoBoostrap());
-        } else {
+        if (instanceState.getRestoreStatus() != null
```

Comment:
I don't see the purpose of this check. Why not just pass the `auto_bootstrap` setting through from the config 100% of the time?

Reply:
Idea is - if we are doing a restore then we need to set auto_bootstrap to `false` else use the provided value in the configuration provider (like when creating a new cluster with `false` or `true` for most of the cases). Since we can choose to restore to bypass Cassandra streaming we need to override the configured value. I wanted to keep that logic in tuner instead of putting in a configuration provider.

Reply:
Can't we leave `auto_bootstrap=true`, even when we restore?